src/components/Header/AuthenticatedHeaderOption.js

```diff
@@ -0,0 +1,464 @@
+  const handleCloseSwitchView = () => {
+    setDropDown(null);
+  };
+
+  const handleOpenUserMenu = (event) => {
+    setAnchorElUser(event.currentTarget);
+  };
+
+  const handleCloseUserMenu = () => {
+    setAnchorElUser(null);
+  };
+
+  return (
+    <>
+      <Box
+        sx={{
+          flexGrow: 1,
+          display: {
+            xs: leftDrawer ? "block" : "none",
+            md: leftDrawer ? "none" : "flex",
+          },
+        }}
+      >
+        {(switchView === "student" || merakiStudents || studentView) && (
+          <>
+            <Box
+              sx={{
+                flexGrow: 1,
+                display: {
+                  xs: "none",
+                  md: "flex",
+                },
+              }}
+            >
+              <MenuItem onClick={handleOpenLearn}>
+                Learn
+                {learn ? <ExpandLessIcon /> : <ExpandMoreIcon />}
+              </MenuItem>
+              <DropDown
+                dropDown="Learn"
+                indicator={learn}
+                handleClose={handleCloseLearn}
+                toggleDrawer={toggleDrawer}
+              />
+              <MenuItem onClick={toggleDrawer && toggleDrawer(false)}>
+                <NavLink
+                  to={PATHS.CLASS}
+                  className={classes.link}
+                  activeClassName={classes.active}
+                >
+                  Classes
+                </NavLink>
+              </MenuItem>
+              <MenuItem onClick={toggleDrawer && toggleDrawer(false)}>
+                <NavLink
+                  to={PATHS.MENTOR}
+                  className={classes.link}
+                  activeClassName={classes.active}
+                >
+                  Mentor
+                </NavLink>
+              </MenuItem>
+            </Box>
+            <Box
+              sx={{
+                display: { xs: "block", md: "none" },
+              }}
+            >
+              <MobileDropDown
+                Menu="Learn"
+                handleClose={handleCloseLearn}
+                toggleDrawer={toggleDrawer}
+              />
+              <MenuItem onClick={toggleDrawer && toggleDrawer(false)}>
+                <NavLink
+                  to={PATHS.CLASS}
+                  className={classes.link}
+                  activeClassName={classes.active}
+                >
+                  Classes
+                </NavLink>
+              </MenuItem>
+              <MenuItem onClick={toggleDrawer && toggleDrawer(false)}>
+                <NavLink
+                  to={PATHS.MENTOR}
+                  className={classes.link}
+                  activeClassName={classes.active}
+                >
+                  Mentor
+                </NavLink>
+              </MenuItem>
+            </Box>
+            <Box
+              sx={{
+                display: { xs: "block", md: "flex" },
+                justifyContent: !isActive && "flex-end",
+                width: !isActive && "100%",
```

Comment:
I'm assuming `md` should be `sm` here since `justifyContent` is set to `flex-end` when the minimum width is `breakpoints.values.sm`.
```suggestion
                display: { xs: "block", sm: "flex" },
                justifyContent: { xs: "normal", sm: "flex-end" },
                width: { xs: 0, sm: "100%" },
```